packages/tldraw/src/lib/ui/context/actions.tsx

```diff
@@ -1510,6 +1510,21 @@ export function ActionsProvider({ overrides, children }: ActionsProviderProps) {
 					trackEvent('change-page', { source, direction: 'next' })
 				},
 			},
+			{
+				id: 'adjust-shape-styles',
+				label: 'a11y.adjust-shape-styles',
+				kbd: 'cmd+Enter,ctrl+Enter',
+				readonlyOk: true,
```

Comment:
Don't think we even show the style menu in readonly?

Reply:
good point! fixed